src/private/doctor/js/registerVisitScript.js

```diff
@@ -0,0 +1,155 @@
+/**
+ * Tutaj nie ruszamy
+ */
+document.addEventListener('DOMContentLoaded', function() {
+    main();
+});
+
+/**
+ * Główna funkcja, tutaj dodajemy skrypty które mają się wywołać po załadowaniu strony
+ */
+function main() {
+    initializeMaterializeSelectors();
+    initializeDatepickers();
+    initializeTimepickers();
+}
+
+/**
+ * Register new appointment by sending data to API
+ */
+function registerAppointment(user) {
+    const submittedData = {};
+
+    // Make a timestamp with date and time and add it to json
+    let dateTimestampInSeconds = convertDateToTimestamp(document.getElementById("appointment-date").value);
+    let timepickerValue = document.getElementById("appointment-time").value;
+    if (!timepickerValue) {
+        alert("Wybierz godzinę wizyty");
+        return;
+    }
+
+    let timeInSeconds = convertTimepickerValueToSeconds(timepickerValue);
+    if (!validateTimepickerValue(timeInSeconds)) {
+        return;
+    }
+    let dateTimeInSeconds = dateTimestampInSeconds + timeInSeconds;
+    submittedData.startDate = dateTimeInSeconds;
+
+    let selectDuration = document.getElementById("select-duration");
+    let appointmentDuration = selectDuration.options[selectDuration.selectedIndex].value;
+    // Appointment duration should be presented in seconds
+    if (appointmentDuration == 1) {
+        submittedData.duration = 3600;
+    } else if (appointmentDuration == 2) {
+        submittedData.duration = 7200;
+    } else submittedData.duration = 0;
+
+    // Doctor email
```

Comment:
Patient email, ale z tego co patrzę to błąd tylko w komentarzu na szczęście

Reply:
Poprawione